### PR TITLE
Update for Flarum v1.0

### PR DESCRIPTION
**Pull request by iPurpl3x**
**Changes proposed in this pull request:**

Updated composer.json to support Flarum v1.0.0

### CODE REVIEW

File: js/src/forum/components/CreatePollModal.js [hunk 1]
```diff
@@ -109,8 +112,9 @@ export default class CreatePollModal extends Modal {
     }
 
     displayOptions() {
+        console.log(this.options);
```
Comment (imorland) [inline js/src/forum/components/CreatePollModal.js:115]: Whoops 🙈

File: js/src/forum/components/CreatePollModal.js [hunk 2]
```diff
@@ -109,8 +112,9 @@ export default class CreatePollModal extends Modal {
...
         return Object.keys(this.options).map((el, i) => (
-            <div className={this.options[i + 1] === '' ? 'Form-group hide' : 'Form-group'}>
+            <div className={classList('Form-group', this.options[i + 1]() === '' && 'hide')}>
```
Comment (imorland) [inline js/src/forum/components/CreatePollModal.js:117]: console error here

image

**Review by imorland — CHANGES_REQUESTED** [review]

File: js/src/forum/components/CreatePollModal.js (hunk 1 above)
Comment (iPurpl3x) [reply js/src/forum/components/CreatePollModal.js]: 🙈 
I have removed that now.

File: js/src/forum/components/CreatePollModal.js (hunk 2 above)
Comment (iPurpl3x) [reply js/src/forum/components/CreatePollModal.js]: I had that error while developing then I thought that I resolved it but I guess my changes got lost into the void...
Now I tried to understand why this is there anyway and found out that this case actually never exists (` this.options[i + 1]() === ''` so I just removed it. Also in EditPollModal.js it's not present so I guess that it isn't needed. 

Maybe @datitisev knows better here ?

**Pull request merged**